src/components/Post.js

```diff
@@ -3,81 +3,81 @@ import axiosInstance from "../api/axios";
 import img1 from "../images/img1.jpg";
 import { useNavigate } from "react-router-dom";
 
-
 const Post = () => {
   const [blog, setBlog] = useState({ title: "", content: "", author: "" });
-  let navigate = useNavigate();
 
-  const getBlogDetails = (e) => {
+  const handleBlogDetails = (e) => {
     e.preventDefault();
     setBlog((current) => ({ ...current, [e.target.name]: e.target.value }));
   };
 
+  let navigate = useNavigate();
   const handleSubmit = async (e) => {
     e.preventDefault();
     try {
-        const result = axiosInstance.post({setBlog});
-      if (result?.status === 200) {
-        return navigate("/", {replace: true});
-      }
+        await axiosInstance.post({ setBlog });
+
     } catch (err) {
       console.log(err);
     }
+    navigate("/blogs");
```

Comment:
You want to redirect to `/blogs` even if there's an error?

Reply:
I have moved to the try block and created and error message in the catch block for this